nixos/modules/services/games/asf.nix

```diff
@@ -156,6 +160,7 @@ respectively `0` because NixOS takes care of updating everything.
     users = {
       users.asf = {
         home = cfg.dataDir;
+        homeMode = "700";
```

Comment:
Does this also make the config.ASF.json (which is created by us) read only for the asf user? Currently it's readable by anyone (not a great idea because of the IPC password, even with this change)

Reply:
That is created in the preStart section of the systemd unit? I think that is running as the asf user. I chmod the directory to 700 already and didn't find any regressions.